src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/ReviewUnusedParameters.Fixer.cs

```diff
@@ -23,8 +31,144 @@ public sealed override FixAllProvider GetFixAllProvider()
         public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
         {
             // Fixer not yet implemented.
+            Diagnostic diagnostic = context.Diagnostics.Single();
+
+            context.RegisterCodeFix(
+                new MyCodeAction(
+                    MicrosoftMaintainabilityAnalyzersResources.ReviewUnusedParametersMessage,
```

Comment:
We generally define a separate, concise title string for the code fix. Possibly something line "Remove unused parameter"?